research/neural_gpu/neural_gpu_trainer.py

```diff
@@ -552,7 +553,7 @@ def score_beams_prog(beams, target, inp, history, print_out=False,
                 for h in history]
   tgt_set = set(target)
   if print_out:
-    print "target: ", tgt_prog
+    print("target: ", tgt_prog)
```

Comment:
This needs `from __future__ import print_function` above.

Reply:
Thanks!  Done.